jobs/integration/validation.py

```diff
@@ -761,7 +767,6 @@ async def test_gpu_support(model, tools):
         control_plane,
         "ds",
         ["nvidia-device-plugin-daemonset"],
-        "-n kube-system",
```

Comment:
AHH yes -- because it's now in the namespace created by juju.  

you could make this `f"-n {k8s_model.name}"` because it should be deployed in there.  But you should be good to go with what you have